ts/Dashboard/Actions/Bindings.ts

```diff
@@ -98,6 +99,14 @@ class Bindings {
                         })
                     )
                     break;
+                case 'threshold':
```

Comment:
Can you add it to the `fromJSON` method too (if it will not be broken like in regular KPI is)?

Reply:
We decided to see later how the Threshold component works. So for now I'm going to remove this change and in the future, we can discover and decide how it should look like